src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.text.ParseException;
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ConcurEventNotificationDTO;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationConversionService;
+
+public class ConcurEventNotificationConversionServiceImpl implements ConcurEventNotificationConversionService {
+    
+    protected DateTimeService dateTimeService;
+
+    @Override
+    public ConcurEventNotification convertConcurEventNotification(ConcurEventNotificationDTO concurEventNotificationDTO) {
+        ConcurEventNotification concurEventNotification = new ConcurEventNotification();
+        concurEventNotification.setContext(concurEventNotificationDTO.getContext());
+        try {
+            concurEventNotification.setEventDateTime(dateTimeService.convertToSqlDate(concurEventNotificationDTO.getEventDateTime()));
+        } catch (ParseException e) {
+            concurEventNotification.setEventDateTime(dateTimeService.getCurrentSqlDate());
```

Comment:
Should this maybe log an error or warning that the event date time was in an invalid format? And we're ok with defaulting to today?